modules/nw-operator-cr.adoc

```diff
@@ -0,0 +1,138 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-network-operator.adoc
+// * installing/installing_aws/installing-aws-network-customizations.adoc
+
+ifeval::["{context}" == "installing-aws-network-customizations"]
+:install:
+endif::[]
+ifeval::["{context}" != "installing-aws-network-customizations"]
+:not-install:
+endif::[]
+
+// Extract parameter descriptions that may have a different ordinal
+// position depending on the module context.
+
+:kube-proxy: pass:q[The parameters in this section specify the `kube-proxy` \
+configuration. If you do not specify the parameter values, the \
+Network Operator applies the default parameter values.]
+
+:kube-proxy-ip: pass:q[The IP address for `kube-proxy` to serve on. \
+The default value is `0.0.0.0`.]
+
+:kube-proxy-refresh: pass:q[The refresh period for `iptables` rules. The default \
+value is `30s`. Valid suffixes include `s`, `m`, and `h` and are described in \
+the link:https://golang.org/pkg/time/#ParseDuration[Go time package] \
+documentation.]
+
+:iptables-min-sync-period: pass:q[The minimum duration before refreshing `iptables` \
+rules. This parameter ensures that the refresh does not happen too frequently. \
+Valid suffixes include `s`, `m`, and `h` and are described in the \
+link:https://golang.org/pkg/time/#ParseDuration[Go time package]]
+
+// Begin module
+
+[id="network-networkoperator-default-cr-{context}"]
+= Network Operator Custom Resource (CR)
+
+The network configuration in an `Network.operator.openshift.io` Custom Resource
+(CR) stores the configuration settings for the Network Operator.
+
+ifdef::install[]
+The following parameters are set in the `install-config.yaml` file:
```

Comment:
So, are you saying that you need to make sure that the parameter values that you set in the install-config.yaml file are also entered into this CR? If so, I'd say:

> The following parameter values must match the values that you set in the `install-config.yaml` file.

I actually think that this might be clear enough in the annotations.